x/uniswap/internal/types/params.go

```diff
@@ -0,0 +1,64 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/params"
+)
+
+// TODO: come up with better naming for these params?
+// Parameter store keys
+var (
+	ParamStoreKeyFeeParams = []byte("feeparams")
+)
+
+// uniswap parameters
+// Fee = 1 - (FeeN - FeeD)
+type FeeParams struct {
+	FeeN sdk.Int `json:"fee_numerator"`   // fee numerator
```

Comment:
Why don't you just use `sdk.Dec` here?

Reply:
The way getInputAmont/getOutputAmount are constructed in keeper.go requires these params. Would love help on reworking the equations to use the fee as sdk.Dec

Reply:
Agree, let’s use Dec